### ISSUE TITLE
Feature: Add FAQ or instructions page and remove some text from the home page , it's too wordy

### ISSUE THREAD

**Issue by m-oniqu3** [SchoolyB/International-Data-Matrix#247]

**Comment by SchoolyB** [comment]
@m-oniqu3 what do you think of this https://github.com/SchoolyB/International-Data-Matrix/blob/client-side/clean-homepage-add-FAQ-247/client/src/pages/Faq.tsx
![faq1](https://github.com/SchoolyB/International-Data-Matrix/assets/110133622/62a5abda-b790-4dda-b190-042eafd52a7a)
![faq2](https://github.com/SchoolyB/International-Data-Matrix/assets/110133622/26748928-cf7c-4933-bd4e-e38c2140263f)

**Comment by SchoolyB** [comment]
Also removed some text from home page  👍🏾

**Comment by SchoolyB** [comment]
See #253

**Comment by SchoolyB** [comment]
Adding this feature as well as fixing #246 makes this
![oops](https://github.com/SchoolyB/International-Data-Matrix/assets/110133622/3dbfdc3e-ae6e-43f5-8183-77d9133a5853)

**Comment by SchoolyB** [comment]
![better](https://github.com/SchoolyB/International-Data-Matrix/assets/110133622/d20ca3ab-10fa-4c55-ab60-7541be84c0e8)
